services/catalog/service/migrations/committed/000014-channel-localizations-and-playlist.sql

```diff
@@ -0,0 +1,86 @@
+--! Previous: sha1:7450cde0cb372030062d77be9527127598e1180f
+--! Hash: sha1:a6c702858a035cb36f13575f70a7bc19528e7500
+--! Message: channel-localizations-and-playlist
+
+-- channel localizations
+DROP TABLE IF EXISTS app_public.channel_localizations CASCADE;
+CREATE TABLE IF NOT EXISTS app_public.channel_localizations(
+  id INTEGER PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+  channel_id TEXT REFERENCES app_public.channel ON DELETE CASCADE,
+  locale TEXT NOT NULL,
+  is_default_locale BOOLEAN NOT NULL,
+  title TEXT NOT NULL,
+  description TEXT
+);
+SELECT ax_define.define_index('channel_id', 'channel_localizations', 'app_public');
+SELECT ax_define.define_index('locale', 'channel_localizations', 'app_public');
+
+ALTER TABLE app_public.channel DROP COLUMN IF EXISTS title;
```

Comment:
Since there is no migration of existing entities to the channel_localizations
table - I assume it's fine and we don't need to mentione any migration notes in
the release notes, from Managed to Customizable, correct?

Reply:
Channels have not much data so they can easily be recreated. And playlist must anyways be recreated for every day so there is no automatic migration planned.